backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -65,37 +66,41 @@
     return staffService.getStaffEngagementsBetween(user, start, end);
   }
 
+  private void verifyManagementRole(String authString) {
```

Comment:
In a future PR this could also be moved to `AuthenticationUtils` or `StaffService`.

Reply:
I kept it here because all this function does is throw ResponseStatusException, which in my view is controller's responsibility.
This keeps AuthenticationUtils and StaffService independent of any http status.

Reply:
Yep cool!